enhancements/sig-architecture/98-managed-rbac/README.md

```diff
@@ -0,0 +1,185 @@
+including Role, ClusterRole, RoleBinding, ClusterRoleBinding.
+A valid `ManagedRBAC` resource should be in a "cluster namespace" and
+the associated RBAC resources will be deliver to the associated managed cluster
+with that "cluster namespace".
+
+## Motivation
+
+### Distribute and manage multi-cluster RBAC resources
+
+The `ManagedRBAC` will help us easily distribute
+RBAC resources to the managed cluster. The `ManagedRBAC` controller using
+`ManifestWork` API will ensuring, updating, removing the RBAC resources
+from each of the managed cluster. 
+The `ManagedRBAC` API will protect these distributed RBAC resources from unexpected 
+modification and removal.
+
+### Enhance the usability of the existing ManagedServiceAccount API
+
+After ensuring the service-accounts' presence on each managed cluster using the existing
+[ManagedServiceAccount](https://github.com/open-cluster-management-io/managed-serviceaccount)
+API, the `ManagedRBAC` can reference the `ManagedServiceAccount` and use that
+as the Subject for the RBAC resources. This will help regulate the access level for
+the `ManagedServiceAccount`.
+
+### New API instead of just using ManifestWork API
+
+There are several reasons why we should create a new API instead of using `ManifestWork`:
+- `ManifestWork` API is meant to be a multicluster workload delivery primitive type.
+- `ManagedRBAC` API is a higher level concept that users can understand its multicluster
+RBAC resources scope easily.
+- `ManifestWork` API payload does not have RBAC specific fields definitions.
+- `ManagedRBAC` can determine the `ManagedServiceAccount` subject and create `ManifestWork`
+with the correct RoleBinding/ClusterRoleBinding subject.
+
+## Goals & Non-Goals
+
+### Goals
+
+- Manage the lifecycle of RBAC resources across multiple cluster.
+- Allows the typical Subject Kind: Group, ServiceAccount, and User
+for RoleBinding and ClusterRoleBinding.
+- Allows the `ManagedServiceAccount` as Subject Kind 
+for RoleBinding and ClusterRoleBinding.
+
+### Future goals
+
+- It is currently assumed that the user of `ManagedRBAC` is either a cluster admin or
+a user who can create `ManagedRBAC` in the hub cluster's managed "cluster namespace".
+It will be useful if there is an approval process that a cluster admin can approve,
+which will allow for other users to create `ManagedRBAC` resources.
+- Support creating a RoleBinding/ClusterRoleBinding without defining any Role/ClusterRole
+rules.
+
+## Design
+
+### Component & API
+
+We purpose to adding a new custom resource named
+`ManagedRBAC` introduced into OCM by this proposal:
+
+A sample of the `ManagedRBAC` will be:
+
+```yaml
+apiVersion: rbac.open-cluster-management.io/v1alpha1
+kind: ManagedRBAC
+metadata:
+  name: my-managed-rbac
+  namespace: cluster1
+spec:
+  clusterRole:
+    rules:
+    - apiGroups: ["apps"]
+      resources: ["deployments"]
+      verbs: ["update"]
+  roles:
+  - namespace: default
```

Comment:
how about a namespace selector? What if you want to multiple namespaces to have the same rule?